lib/dma.c

```diff
@@ -195,8 +180,9 @@ dma_controller_remove_regions(dma_controller_t *dma)
     for (i = 0; i < dma->nregions; i++) {
         dma_memory_region_t *region = &dma->regions[i];
 
-        vfu_log(dma->vfu_ctx, LOG_INFO, "unmap vaddr=%p IOVA=%lx",
-               region->virt_addr, region->dma_addr);
+        vfu_log(dma->vfu_ctx, LOG_INFO, "unmap IOVA=%#lx-%#lx) vaddr=%#lx-%#lx)",
```

Comment:
this isn't the right syntax for the ranges? Should be [...), and then you don't want to take 1 off, e.g. [0-4096) is 0, 1, 2,... 4095. Or if you prefer to keep the -1, it should be [0-4095]

Reply:
Right.